Underscore.cs/List/Contract/Partition.cs

```diff
@@ -51,22 +51,22 @@ public interface IPartitionComponent
         /// from the back of the list
         /// </summary>
         /// <typeparam name="T">The type of the elements in the list</typeparam>
-        /// <param name="list">The list to take the slice from</param>
-        /// <param name="start">The start index</param>
-        /// <param name="end">The end index</param>
+        /// <param name="start">The inclusive start index</param>
+        /// <param name="end">The inclusive end index</param>
+        /// <returns>slice of the list</returns>
         IList<T> Slice<T>( IList<T> list, int start, int end );
 
 
+
         /// <summary>
         /// Takes a slice from a list, if start is greater then the end index
         /// the results are reversed, if the index is negative corresponds to the index
         /// from the back of the list, if the slice is larger than the size of the list
         /// then the items are repeated
         /// </summary>
         /// <typeparam name="T">The type of the elements in the list</typeparam>
-        /// <param name="list">The list to take the slice from</param>
-        /// <param name="start">The start index</param>
-        /// <param name="end">The end index</param>
+        /// <param name="start">The inclusive start index</param>
+        /// <param name="end">The inclusive end index</param>
```

Comment:
I think this should be exclusive -- it's the standard for subarray/substring operations in pretty much every language (including python's slice)